SitumWayfinding/Localizations/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,116 @@
+
+/* Button to center map in current location of user */
+"positioning.center" = "Center";
+
+/* No comment provided by engineer. */
+"positioning.createMarker" = "Create marker";
+
+/* Shown to user when select a destination (destination is any free point that user selects on the map) */
+"positioning.customDestination" = "Custom destination";
+
+/* Alert message to show to the user when destination was reached */
+"positioning.destinationReached.alert.message" = "You've arrived to your destination";
+
+/* Alert title to show to the user when destination was reached */
+"positioning.destinationReached.alert.title" = "Destination Reached";
+
+/* Alert title error when download the floor plan fails */
+"positioning.error.emptyFloor.alert.message" = "An unexpected error ocurred while downloading the floorplan. Please try again.";
+
+/* Alert title error when download the floor plan fails */
+"positioning.error.emptyFloor.alert.title" = "Empty floorplan";
+
+/* Alert title error when the user does not select a destination */
+"positioning.error.noDestinationSelected.alert.title" = "No destination selected";
+
+/* Alert title error when the user is outdoor */
+"positioning.error.positionOutdoor.alert.title" = "Position outdoor";
+
+/* Alert title error when the user position is unknown */
+"positioning.error.positionUnknown.alert.title" = "Position unknown";
+
+/* Alert title error when WayFinding cannot compute route to destination */
+"positioning.error.unableToComputeRoute.alert.title" = "Unable to compute route";
+
+/* Alert message to show for a long press action */
+"positioning.longPressAction.alert.message" = "Select an action:";
+
+/* Alert title to show for a long press action */
+"positioning.longPressAction.alert.title" = "Long press actions";
+
+/* Show to the user distance to next point */
+"positioning.navigationProgress" = "%.1fm remaining";
+
+/* Used to show the user the number of beacons that library detects nearby */
+"positioning.numBeacons" = "%d beacons";
+
+/* Alert message to show user is outside of route */
+"positioning.outsideBuilding.alert.message" = "The user is currently outside of the building. Positioning will resume when the user returns.";
+
+/* Alert title to show user is outside of building */
+"positioning.outsideBuilding.alert.title" = "User outside building";
+
+/* Alert title to show user is outside of route */
+"positioning.outsideRoute.alert.title" = "User outside route";
+
+/* Placeholder for searching pois */
+"positioning.searchPois" = "Search Pois";
+
+/* Error when user does not set credentials */
+"situmMapsLibrary.error.credentials" = "You must set your auth info with setCredentials() before calling load(buildingWithId:)";
+
+/* Error when user does not set correctly building id */
+"situmMapsLibrary.error.invalidBuilding" = "The building ID in plist file is not correct. Please add a valid ID with key: es.situm.sdk.ACTIVE_BUILDING_ID";
```

Comment:
The builidng ID is not correct. Please add a valid building ID in LibrarySettings